android/sdk/src/main/java/org/jitsi/meet/sdk/AudioDeviceHandlerConnectionService.java

```diff
@@ -152,7 +157,7 @@ public void start(AudioModeModule audioModeModule) {
 
     @Override
     public void stop() {
```

Comment:
I guess it also makes sense to set the context to null here, just to make sure there is leak.